### DISCUSSION TITLE
BlueSCSI working in a PC-XT Clone, with Seagate ST-02 controller card

### DISCUSSION THREAD

**Discussion by pivanow** [erichelgeson/BlueSCSI-v1#233] [General]
Tested and working fine in a PC-XT Clone, with Seagate ST-02 controller card.
![PC-XT and Seagate ST-02](https://user-images.githubusercontent.com/131489199/233722538-7c61acca-c4cf-493b-9442-38e57c3e51f4.jpeg)

**Comment by erichelgeson** [comment]
Thanks! I've added it to the https://github.com/erichelgeson/BlueSCSI/wiki/Supported-Systems page
